recipes/gdbm/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "gdbm" %}
+{% set version = "1.18" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: ftp://ftp.gnu.org/gnu/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: e316f8e4a3e7e4f23955be65d54fec48
```

Comment:
We should probably find a better checksum if there is one.

Reply:
we could compute it ourselves. there is a sha1 on the website too.

Reply:
They also have a pgp signature, doese that work? https://www.gnu.org.ua/software/gdbm/download.html

Reply:
Don't think we can use PGP. Might be worth raising to `conda-build`. Not sure how easy this would be to implement in practice.

SHA1 should be fine for now. Will update.